ecep_cms/docker-compose.dev.yml

```diff
@@ -0,0 +1,56 @@
+version: "3"
+services:
+  cms:
+    build: .
+    ports:
+      - 127.0.0.1:8000:8000
+    command: ["python3", "manage.py", "runserver", "0.0.0.0:8000"]
+    container_name: core_cms
+    hostname: core_cms
+    volumes:
+      - ./src/apps:/code/apps
+      - ./src/taccsite_custom:/code/taccsite_custom
+      - ./src/taccsite_cms/custom_app_settings.py:/code/taccsite_cms/custom_app_settings.py
+      - ./src/taccsite_cms/urls_custom.py:/code/taccsite_cms/urls_custom.py
+      - ./src/taccsite_cms/settings_custom.py:/code/taccsite_cms/settings_custom.py
+      - ./src/taccsite_cms/settings_local.py:/code/taccsite_cms/settings_local.py
+      - ./src/taccsite_cms/secrets.py:/code/taccsite_cms/secrets.py
+    networks:
+      - core_cms_net
+
+  postgres:
```

Comment:
We are using the external WMA DB VM for all portals and SAD CMS setups now.
And AFAIK we are using the external ES cluster for all search as well.
I think we can drop both the DB container and the ES container configs across all project docker-compose.yml files.

Reply:
If I delete `postgres:` and `elasticsearch:` then what code would tell those containers to be created?

<details><summary>Wes Tries to Answer Himself</summary>

Ah, Core-CMS `./docker-compose.dev.yml`… by having these ones extend it e.g.

```
  postgres:
    extends:
      file: docker-compose.dev.yml
      service: postgres
```

But then, how do I access that? It **is** in the `core_cms` container, but it it is **not** in the repo. So… I would sync the container `/code/docker-compose.dev.yml` to local repo clone `/docker-compose.core-cms.yml`, e.g.

```
    volumes:
      - ./docker-compose.core-cms.yml:/code/docker-compose.dev.yml
```

And then:

```
  postgres:
    extends:
      file: ../docker-compose.core-cms.yml
      service: postgres
```

**Correct?**

</details>